### ISSUE TITLE
Suggested redesign of Connect screen

### ISSUE THREAD

**Issue by rikedyp** [Dyalog/ride#1190]
### Describe the issue you are having
The new layout is less overwhelming, but it is still not 100% clear what the edit button does.

This mockup suggests a new layout. The play button to the immediate left of the configuration name is more obvious that clicking it will start that configuration, and the three-dot burger menu is a more universal "edit" or "options" icon than the current one.

![image](https://github.com/Dyalog/ride/assets/9338369/87f08acd-24fa-4d6c-932b-229d50ceb488)

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: 4.5.4043
  Electron: 25.3.2
  Chrome: 114.0.5735.248
  Node: 18.15.0
  Platform: Win32
  Date: 2023-09-26 17:25:50 +0200
  Git commit: 5f3a6111e4c67f4c605b4cdbaff1088208d5972d
  Preferences:{
    "defaultConfig":"v19.0",
    "kbdLocale":"en_GB"
  }

Interpreter:
  Version: 19.0.48508
  Platform: Windows-64
  Edition: Unicode/64
  Date: Jan 13 2024 at 20:09:17

```

**Comment by abrudz** [comment]
I like the idea, but I don't think the `⁝` or `☰` works well when they don't open a drop-down menu. However, a gear icon could work.

**Comment by abrudz** [comment]
Closed in favour of #1197
